<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SniffWindow</class>
 <widget class="QMainWindow" name="SniffWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="layout_filter">
      <item>
       <widget class="QLabel" name="lb_filter">
        <property name="text">
         <string>F&amp;ilter:</string>
        </property>
        <property name="buddy">
         <cstring>tb_filter</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="History_Line_Edit" name="tb_filter">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_filter_clear">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_filter_apply">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableView" name="table_packets">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTreeWidget" name="tree_packet">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QHexEdit" name="hexEdit">
       <property name="font">
        <font>
         <family>Sans</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Hex View of Packet</string>
       </property>
       <property name="whatsThis">
        <string>Hex View of Packet</string>
       </property>
       <property name="readOnly" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_recent_files">
     <property name="title">
      <string>Recent Files</string>
     </property>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menu_recent_files"/>
    <addaction name="action_save_all"/>
    <addaction name="action_save_shown"/>
    <addaction name="action_preferences"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_Sniff">
    <property name="title">
     <string>&amp;Sniff</string>
    </property>
    <addaction name="actionStop"/>
    <addaction name="actionSniff"/>
    <addaction name="separator"/>
    <addaction name="action_remove_all"/>
    <addaction name="action_remove_shown"/>
    <addaction name="separator"/>
    <addaction name="actionDisableOptions"/>
   </widget>
   <widget class="QMenu" name="menuStats">
    <property name="title">
     <string>S&amp;tats</string>
    </property>
    <addaction name="actionTable"/>
    <addaction name="action_only_Shown"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Table"/>
    <addaction name="action_Tree"/>
    <addaction name="action_Hex"/>
    <addaction name="action_Python"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Sniff"/>
   <addaction name="menuStats"/>
   <addaction name="menu_View"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSniff"/>
   <addaction name="actionStop"/>
  </widget>
  <widget class="StatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_save_all">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/capture_stop.png</normaloff>:/icons/capture_stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionSniff">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/capture_start.png</normaloff>:/icons/capture_start.png</iconset>
   </property>
   <property name="text">
    <string>Sni&amp;ff</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionTable">
   <property name="text">
    <string>&amp;Table</string>
   </property>
  </action>
  <action name="action_remove_all">
   <property name="text">
    <string>Remove &amp;All</string>
   </property>
  </action>
  <action name="actionDisableOptions">
   <property name="text">
    <string>Disable &amp;Background Tasks</string>
   </property>
  </action>
  <action name="action_Table">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Table</string>
   </property>
  </action>
  <action name="action_Tree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>T&amp;ree</string>
   </property>
  </action>
  <action name="action_Hex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Hex</string>
   </property>
  </action>
  <action name="action_Python">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Python</string>
   </property>
  </action>
  <action name="action_remove_shown">
   <property name="text">
    <string>Remove &amp;Shown</string>
   </property>
  </action>
  <action name="action_save_shown">
   <property name="text">
    <string>Save &amp;Shown</string>
   </property>
  </action>
  <action name="action_only_Shown">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Only &amp;Shown</string>
   </property>
  </action>
  <action name="action_preferences">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/preferences.png</normaloff>:/icons/preferences.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QHexEdit</class>
   <extends>QScrollArea</extends>
   <header>QHexEdit/qhexedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>History_Line_Edit</class>
   <extends>QLineEdit</extends>
   <header>widgets/history_line_edit.h</header>
  </customwidget>
  <customwidget>
   <class>StatusBar</class>
   <extends>QStatusBar</extends>
   <header>statusbar.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>SniffWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tb_filter</sender>
   <signal>returnPressed()</signal>
   <receiver>bt_filter_apply</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>746</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
